hw/top_earlgrey/fpv/top_earlgrey_fpv_cfgs.hjson

```diff
@@ -0,0 +1,148 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+
+  // This is the master cfg hjson for FPV. It imports ALL individual FPV
+  // cfgs of the IPs and the full chip used in top_earlgrey. This enables to run
+  // them all as a regression in one shot.
+  name: top_earlgrey_batch_fpv
+
+  import_cfgs: [// common server configuration for results upload
+                "{proj_root}/hw/data/common_project_cfg.hjson"]
+
+  use_cfgs: [// TODO: implement some switch to only select "_fpv" testbenches
+             // TODO: if we default "_fpv" cov to be on, and the rest of the tbs cov off, need a
+             // command-line switch to disable cov.
+             {
+               name: prim_arbiter_ppc_fpv
+               fusesoc_core: lowrisc:fpv:prim_arbiter_ppc_fpv
+               import_cfgs: ["{proj_root}/hw/formal/data/common_fpv_cfg.hjson"]
```

Comment:
@msfschaffner you commented here:
can we still override the tool setting on the command line if we set it here? we potentially want to use this with multiple formal tools, hence we should make sure this does not prevent us from doing that.

In the lint case for instance I am using the same batch cfg for both ascentlint and verible...

My answer is: yes, we can still override the `tool` option by setting the `-t` on the command line. However, I think we might not need to force user to set `-t`: https://github.com/lowRISC/opentitan/blob/master/util/dvsim/OneShotCfg.py#L26

Reply:
ok, sounds good - just wanted to check that this is still possible.